src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -74,10 +82,15 @@ public function __construct(MigrationRepositoryInterface $repository,
      *
      * @param  array|string  $paths
      * @param  array  $options
+     * @params Symfony\Component\Console\Output\OutputInterface $output
      * @return array
      */
-    public function run($paths = [], array $options = [])
+    public function run($paths = [], array $options = [], OutputInterface $output = null)
```

Comment:
Changes to method signatures can break existing installations. Should this target the master branch?

Reply:
existing installations doesn't break with these changes and the end result is same. Anyways third parameter is nullable and it would work same as before if the third parameter is not passed.